manifests/init.pp

```diff
@@ -82,6 +86,8 @@
   validate_absolute_path($config_path)
   validate_bool($config_cluster)
   validate_bool($config_stomp)
+  validate_bool($config_mqtt)
+  validate_bool($mqtt_ensure)
```

Comment:
The config_mqtt option should be a bool. It determines whether specific config should be written for the plugin. It is perfectly valid to run the plugin without config. It just takes the defaults build into the plugin. See below for mqtt_ensure.